internal/metrics/metrics.go

```diff
@@ -87,33 +87,38 @@ func newStateCounter(ns *gometrics.Namespace, desc *prometheus.Desc) *StateCount
 }
 
 // Get returns the count of containers in running, paused, and stopped states
-func (ctr *StateCounter) Get() (running int, paused int, stopped int) {
+func (ctr *StateCounter) Get() (running, paused, stopped int) {
 	ctr.mu.RLock()
 	defer ctr.mu.RUnlock()
 
-	states := map[string]int{
-		"running": 0,
-		"paused":  0,
-		"stopped": 0,
-	}
+	running, paused, stopped = 0, 0, 0
```

Comment:
Nit: [this line is a no-op](https://go.dev/doc/effective_go#named-results:~:text=When%20named%2C%20they%20are%20initialized%20to%20the%20zero%20values%20for%20their%20types%20when%20the%20function%20begins) and can be removed.